mediachain/indexer/mc_simpleclient.py

```diff
@@ -209,11 +209,11 @@ class SimpleClient(object):
     
     def __init__(self,
                  ## Hard Code for now:
-                 datastore_host = '107.23.23.184',
+                 datastore_host = 'facade.mediachain.io',
                  datastore_port = 10002,
-                 transactor_host = '107.23.23.184',
+                 transactor_host = 'facade.mediachain.io',
```

Comment:
Thanks, I'll make a note to re-add configurability soon. In the next ~week we'll figure out which project this code is going to live in, and re-enable configuration.